plugins/inlinetoolbar/plugin.js

```diff
@@ -80,6 +103,19 @@
 			};
 
 			/**
+			 * Create html representation of menu items in toolbar.
```

Comment:
HTML is an abbreviation, as such it should be uppercased, it's HTML, CSS, YAML etc.